docker/Dockerfile

```
@@ -32,4 +30,4 @@ RUN pipenv install --system --deploy
 COPY . .
 
 # Start the application!
-CMD ["python", "main.py"]
+CMD ["python", "-m", "blackbox"]
```

```suggestion
CMD ["python", "-m", "blackbox"]
```
Maybe we actually use the `blackbox` command here, as a sanity check? Just gotta install it with pip install -e . first.